src/Illuminate/Foundation/Providers/ArtisanServiceProvider.php

```diff
@@ -8,6 +8,7 @@
 use Illuminate\Cache\Console\ForgetCommand as CacheForgetCommand;
```

Comment:
```suggestion
 use Illuminate\Cache\Console\ForgetCommand as CacheForgetCommand;```suggestion
 use Illuminate\Cache\Console\ForgetCommand as CacheForgetCommand;```suggestion
 use Illuminate\Cache\Console\ForgetCommand as CacheForgetCommand;```suggestion
 use Illuminate\Cache\Console\ForgetCommand as CacheForgetCommand;```suggestion
 use Illuminate\Cache\Console\ForgetCommand as CacheForgetCommand;
```

```

```

```

```